accounts/forms.py

```diff
@@ -0,0 +1,8 @@
+from django.contrib.auth.forms import UserCreationForm
+from django.contrib.auth.models import User
```

Comment:
今の状態だと、せっかくカスタムユーザーを作成したのに、既存のUserを使用していることになるので、以下のように変更してください。カスタムユーザーは以下のようにget_user_modelから取得することができます。
```suggestion
from django.contrib.auth import get_user_model

User = get_user_model()
```

Reply:
```
from django.contrib.auth import get_user_model
User = get_user_model()
```
get_user_modelをimportしたらUser = get_user_model()とするのが基本だと思ってもらって大丈夫です。もちろん`get_user_model().objects.all()`としてもカスタムユーザーからユーザー情報は取ってこれるのですが、これだと長いので`User = get_user_model()`としているという感じです。